src/runner.rs

```diff
@@ -82,16 +82,24 @@ impl TestErrorKind {
     }
 }
 
+type Validator = fn(&Vec<String>, &Vec<String>) -> bool;
```

Comment:
pub type? Please also add docstring for this type.

Reply:
addressed.